improve tutorials & service init for sendgrid (#20)
mesg-foundation/docs: tutorials/erc20-transfer-notifications/send-emails-with-sendgrid.md

You can access the final version of the [source code on GitHub](https://github.com/mesg-foundation/docs/tree/master/tutorials/erc20-transfer-notifications/send-email-with-sendgrid).

You can find a more advanced and maintained version of this service here: [Service Email SendGrid](https://github.com/mesg-foundation/service-email-sendgrid)

::: tip
If you haven't installed **MESG Core** yet, you can do so by running the command:

`bash <(curl -fsSL https://mesg.com/install)`

> You can also install it manually by following [this guide](./installation.md#manual-installation).

Reviewer: wrong link

It's time to create our MESG Service. First, open a terminal in your development folder and run the following command:

```bash
mesg-core service init
```

Then, answer the prompts with the following information:

```text
? Enter the output directory: send-email-with-sendgrid
? Select a template to use Javascript (https://github.com/mesg-foundation/template-service-javascript)
```

The command should have created a `send-email-with-sendgrid` folder containing `mesg.yml`, `Dockerfile` files and a boilerplate for the service.  
Leave these files intact; we'll return to them a bit later in this tutorial.

::: tip
You should see a **mesg.yml** and a **Dockerfile** in your service folder which are the fundamental parts of every MESG service.

* **mesg.yml:** A file that contains all of the metadata of your Service. It gives some global descriptions but also includes the tasks and events that the Service can provide.
  
* **Dockerfile:** A file that describes your Docker container and configures the environment for your service to run inside.
:::

## Send emails through MESG tasks

### Configure your task

Let's add the task we want to serve for MESG Core to the `mesg.yml` file.

> First, clean the `mesg.yml` file, keeping only the keys: `name` and `description`. And declare a `sid` for the service. It should look like this:

Reviewer: ```suggestion
First, clean the `mesg.yml` file, keeping only the keys: `name`, `sid` and `description`. Change their values to look like this:
```
